closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -87,20 +89,47 @@
     // The underlying engine for doing the ChaCha20/Poly1305 work
     private ChaChaEngine engine;
 
-    private static NativeCrypto nativeCrypto;
+    private static final NativeCrypto nativeCrypto;
+    private static final Cleaner contextCleaner;
     private long context;
```

Comment:
I thought I had asked you to make this field `final`, but I don't see that comment. Please do so.

Reply:
both of these are final

private static final NativeCrypto nativeCrypto;
private static final Cleaner contextCleaner;

Reply:
I made the comment with respect to line 94:
```
    private long context;
```

Reply:
ok updated